hamlpy/parser/filters.py

```diff
@@ -1,7 +1,13 @@
 from __future__ import unicode_literals
+"""
+Core HamlPy filters.
+
+The implementation of these should match https://github.com/haml/haml/blob/master/lib/haml/filters.rb as closely as
+possible. Where we differ is that we don't compile Stylus, Coffeescript etc into CSS or Javascript - but place the
+content into suitable <style> and <script> that can be transformed later by something like django-compressor.
+"""
```

Comment:
We of course need the `compress` tag to have sass/less/coffee converted during offline compression, e.g.

```haml
- compress js
  :coffee
    foo=bar
- compress css
  :sass
    .mystyle {...}
```